config/webpacker.yml

```
@@ -34,12 +53,15 @@ development:
   <<: *default
   compile: true
 
+  # Verifies that correct packages and versions are installed by inspecting package.json, yarn.lock, and node_modules
+  check_yarn_integrity: true
+
   # Reference: https://webpack.js.org/configuration/dev-server/
   dev_server:
     https: false
-    host: 0.0.0.0
+    host: localhost
     port: 3035
-    public: 0.0.0.0:3035
+    public: localhost:3035
```

```suggestion
    public: 0.0.0.0:3035
```
Because we use Docker we need the dev server to listen on any ip

---

Good catch... for some reason it seemed to be working for me but I refreshed my dev environment a bit and it didn't work properly until this was changed.